app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+import datetime
+import time
+
+
+def create_file_every_second():
+    now = datetime.datetime.now
```

Comment:
```suggestion
    now = datetime.datetime.now()
```

Reply:
If you check the type of `now` you'll see: `<class 'builtin_function_or_method'>`
I believe you forgot to call this method using `()`.